### PR TITLE
New front

### PR DESCRIPTION
**Pull request by zykinwases**
Frontend added

### CODE REVIEW

**Comment by zykinwases** [comment]
needs some fastapi changes before merging

**Comment by weirdfishes9** [comment]
Need to make visible picture upload

**Comment by weirdfishes9** [comment]
Picture upload only after jackalify button click. Thus picture isn't shown after choosing.

File: src/fastapi/templates/main_form.html [hunk 1]
```diff
@@ -1,19 +1,34 @@
...
         {% if video %}
-            <!-- <video controls="controls">
-                <source src="static/{{video}}" type='video/mp4"'>
-                <a href="static/{{video}}">Download</a>.
-            </video> -->
-            <img src="static/{{video}}">
+            <div id="gif">
+                <p>
+                    <a class="downloadLink" download href="static/working/{{video}}">Download GIF</a>
+                    <a class="downloadLink" download href="static/working/{{photo}}">Download jackalified photo</a>
```
Comment (weirdfishes9) [inline src/fastapi/templates/main_form.html:29]: Gif creates before the image. If you want to download picture right after the end of the process you will get json file and only several seconds later you will be able to download png.

**Review by weirdfishes9 — APPROVED** [review]

**Pull request merged**